eclib/JModel.ec

```diff
@@ -374,13 +374,15 @@ abbrev [-printing] VPBLEND_16u16 = VPBLENDW_256.
 abbrev [-printing] VPBLEND_8u32 = VPBLENDD_256.
 
 (* ------------------------------------------------------------------- *)
+op msb (b: W8.t): bool = 128 <= (to_uint b).
```

Comment:
I’m surprised that there is no such function yet in the `W8` theory. There is `ALU.SF_of` that has a weird name but seems to be what you need. It might be worth to define a general `msb` function it the `BitWord` theory.

Reply:
Yes, I was surprised too,  Didn't think of using `ALU.SF_of` though, but long term it may make more sense to add the `msb` function to the `BitWord` theory. I can include that in this PR...

Reply:
That would be nice, thanks.